File: src/main/java/aurocosh/divinefavor/common/integrations/crafttweaker/ActionRemoveMediumRecipe.java
package aurocosh.divinefavor.common.integrations.crafttweaker;

import aurocosh.divinefavor.common.receipes.RecipeLoader;
import crafttweaker.IAction;

public class ActionRemoveMediumRecipe implements IAction {
    private final String path;

    public ActionRemoveMediumRecipe(String path) {
        this.path = path;
    }

    @Override
    public void apply () {
        RecipeLoader.removeRecipe(path);
    }

    @Override
    public String describe () {
        return String.format("Removed medium recipe: %s", path);
    }
}

File: src/main/kotlin/aurocosh/divinefavor/common/integrations/crafttweaker/ActionRemoveMediumRecipe.kt
package aurocosh.divinefavor.common.integrations.crafttweaker

import aurocosh.divinefavor.common.receipes.RecipeLoader
import crafttweaker.IAction

class ActionRemoveMediumRecipe(private val path: String) : IAction {
    override fun apply() {
        RecipeLoader.removeRecipe(path)
    }

    override fun describe(): String {
        return String.format("Removed medium recipe: %s", path)
    }
}
